pinot-common/src/main/java/com/linkedin/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -373,4 +381,8 @@ public Long value() {
   protected abstract M[] getMeters();
 
   protected abstract G[] getGauges();
+
+  protected String getTableName(String tableName){
+     return (_global ? "unknown" : tableName);
```

Comment:
Would be better to have something like `allTables` instead of `unknown`, since you're collapsing all metrics into a single metric.